components/brave_new_tab_ui/components/default/rewards/index.tsx

```diff
@@ -104,6 +106,7 @@ export const RewardsWidget = createWidget((props: RewardsProps) => {
       nextPaymentDate={adsInfo ? adsInfo.nextPaymentDate : 0}
       earningsThisMonth={adsInfo ? adsInfo.earningsThisMonth : 0}
       earningsLastMonth={adsInfo ? adsInfo.earningsLastMonth : 0}
+      earningsRecieved={earningsReceived}
```

Comment:
Nit: Looks like the property name has a typo (`earningsRecieved`).

Reply:
Fixed